spec/recurly/client_spec.rb

```diff
@@ -66,6 +67,20 @@
         }.to raise_error(ArgumentError, "Invalid region type. Expected one of: #{Recurly::Client::API_HOSTS.keys.join(", ")}")
       end
     end
+
+    describe "using a custom keep alive timeout" do
+      let(:keep_alive_timeout) { 10 }
+
+      it "should use a custom base url in EU" do
+        client = Recurly::Client.new(**client_options.merge(region: :eu, keep_alive_timeout:))
```

Comment:
The ruby client needs to support versions of ruby that do not support the Shorthand Hash Syntax. Please update this and line 80:

```suggestion
        client = Recurly::Client.new(**client_options.merge(region: :eu, keep_alive_timeout: keep_alive_timeout))
```

Reply:
Added